js_modules/dagster-ui/packages/ui-core/src/search/SearchResults.tsx

```diff
@@ -159,6 +159,7 @@ const SearchResultItem = React.memo(({isHighlight, onClickResult, result}: ItemP
               <Caption>{assetFilterPrefixString(item.type)}:&nbsp;</Caption>
             )}
             {labelComponents.map((component) => component)}
+            {item.repoPath && <Caption>&nbsp;in {item.repoPath}</Caption>}
```

Comment:
Instead of relying on a non breaking space here we should rely on a flex-gap to make sure the spacing is consistent.

Reply:
good call